boards/waveshare-rp2040-lcd-0-96/Cargo.toml

```
@@ -3,7 +3,7 @@ name = "waveshare-rp2040-lcd-0-96"
 version = "0.4.0"
 authors = ["René van Dorst <opensource@vdorst.com>", "Andrea Nall <anall@andreanal.com>", "TilCreator <contact.github@tc-j.de>", "The rp-rs Developers"]
 edition = "2018"
-homepage = "https://github.com/rp-rs/rp-hal/tree/main/boards/waveshare-rp2040-lcd-0_96"
+homepage = "https://github.com/rp-rs/rp-hal-boards/tree/main/boards/waveshare-rp2040-lcd-0_96"
```

```suggestion
homepage = "https://github.com/rp-rs/rp-hal-boards/tree/main/boards/waveshare-rp2040-lcd-0-96"
```
This one was already broken, but we should fix it while updating.